sentry/src/main/java/io/sentry/SentryReplayOptions.java

```diff
@@ -147,6 +157,14 @@ public void setRedactAllImages(final boolean redactAllImages) {
     this.redactAllImages = redactAllImages;
   }
 
+  public Set<String> getRedactClasses() {
+    return this.redactClasses;
+  }
+
+  public void setRedactClasses(final Set<String> redactClasses) {
```

Comment:
`m`: can we instead opt for maintaining the in-memory list ourselves and just expose the `addClassToRedact(String className)`? Similar to e.g. how it's done with `bundleIds` here: https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/SentryOptions.java#L1833-L1840

Reply:
Yeah, I can adjust it to what ever is the best practice in java.